### ISSUE TITLE
Documentation points to cover

### ISSUE THREAD

**Issue by gwarf** [EGI-Foundation/EGI-website-tests#2]
A few small points are missing in the README:

- Mozilla geckodriver is required and should be installed manually/externally
- Usage instructions are missing (i.e.  like running `cucumber`)

**Comment by gwarf** [comment]
On Mac OS X it [geckodriver](https://github.com/mozilla/geckodriver) can be installed using [homebrew](https://brew.sh/):
```
brew install geckodriver
```

After this I can see that [Firefox](https://www.mozilla.org/firefox/) is required too.

**Comment by gwarf** [comment]
So after all this:
```sh
 ✘ baptistegrenier@Baptistes-MacBook-Pro  ~/code/repos/EGI-website-tests   master  cucumber
Ignoring ffi-1.9.25 because its extensions are not built.  Try: gem pristine ffi --version 1.9.25
@checkin @public @engagement
Feature: Does the EGI Website correctly describe the state of CheckIn?
  Everybody expects CheckIn to be in production, but it is only production for internal users.
  External users are able to enquire about it, even though it's marked as beta

  Scenario: CheckIn is in Beta for the world, but production internally # features/checkin_is_properly_described.feature:6
    Given I am on www.egi.eu                                            # features/step_definitions/stepdefs.rb:8
    When I go to Services                                               # features/step_definitions/stepdefs.rb:15
    Then I see a link to Internal Services                              # features/step_definitions/stepdefs.rb:19
    And I see a link to CheckIn                                         # features/step_definitions/stepdefs.rb:23
    And The page shows that CheckIn for external users is in Beta       # features/step_definitions/stepdefs.rb:30
    Given I am on the services page                                     # features/step_definitions/stepdefs.rb:38
    When I go to internal services                                      # features/step_definitions/stepdefs.rb:43
    Then it has a list of services                                      # features/step_definitions/stepdefs.rb:47
    And one of them is Security                                         # features/step_definitions/stepdefs.rb:54
    And The security item lists CheckIn                                 # features/step_definitions/stepdefs.rb:59
    But the service is not marked in Beta                               # features/step_definitions/stepdefs.rb:63

1 scenario (1 passed)
11 steps (11 passed)
0m19.603s
```

🍾

**Comment by brucellino** [comment]
Merci!
I discovered that as well last night while writing the Travis integration... I'll add a bit on it in the README.

**Comment by brucellino** [comment]
In principle, we should test across several browsers - this can be a parameter in the test. There are different setups for chrome, and other browsers too.

**Comment by gwarf** [comment]
Yes I agree that multi-browser support would be good (at least Firefox and Chrome, that should cover most of our users, but Edge, Safari, and furthermore w3m, lynx and elinks could be added too to be complete).
Thanks for the doc update!

**Comment by brucellino** [comment]
Closed in 5de1331ca865de7719b9a9dd9e4d0025019a1ee4
